contracts/kernel/Kernel.sol

```diff
@@ -28,6 +29,10 @@ contract Kernel is IKernel, KernelStorage, Petrifiable, IsContract, VaultRecover
         }
     }
 
+    function () external payable {
+        require(isDepositable());
```

Comment:
To work around the changes in https://github.com/aragon/aragonOS/pull/401/files#diff-61de883632b625f81d4e28af414a9c90R64, we could also require that the value must be greater than 0 here and in `AragonApp`.